portal-web/docroot/html/taglib/aui/input/page.jsp

```diff
@@ -25,39 +25,118 @@
 </c:if>
 
 <liferay-util:buffer
-	var="labelContent"
+	var="toggleSwitchLabelContent"
 >
 	<c:if test="<%= Validator.isNotNull(label) %>">
-		<c:if test='<%= type.equals("toggle-switch") %>'>
-			<span class="toggle-switch-label">
+		<span class="toggle-switch-label">
+			<liferay-ui:message key="<%= label %>" localizeKey="<%= localizeLabel %>" />
+		</span>
+
+		<c:if test="<%= required && showRequiredLabel %>">
+			<span class="toggle-switch-text">
+				<liferay-ui:message key="required" />
+
+				<aui:icon cssClass="reference-mark text-warning" image="asterisk" markupView="lexicon" />
+			</span>
 		</c:if>
+	</c:if>
+</liferay-util:buffer>
 
-		<liferay-ui:message key="<%= label %>" localizeKey="<%= localizeLabel %>" />
+<liferay-util:buffer
+	var="toggleSwitchStartContent"
+>
+	<label <%= labelTag %>>
```

Comment:
In [Clay Toggle](https://github.com/liferay/clay/blob/master/packages/clay-form/src/Toggle.tsx#L48) there is a `span` here. In fact, there is no `label` element, instead, label in rendered in a [child `span`](https://github.com/liferay/clay/blob/master/packages/clay-form/src/Toggle.tsx#L113).

The change may break things, but I believe the point of these migration tasks it to fully follow Clay 3 markup, even if it breaks compatibility.

Maybe we want to add a `label` around the Clay 3 component, but still we need to add the [outer span](https://github.com/liferay/clay/blob/master/packages/clay-form/src/Toggle.tsx#L48) from the component.

Reply:
In my opinion the source of truth is [here](https://clayui.com/docs/components/toggle-switch/markup.html). Toggle Switch is wrapped in label. The way it is being done in Clay Toggle requires the `input` to have a unique id and `label` having a `for` attribute with the id so they are associated. It looks like neither happens which is incorrect. If we wrap the `input` in the `label` the association is implicit. See https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label.

Reply:
@matuzalemsteles @bryceosterhaus could you please weigh in on this? Should we 
1. Open a Clay issue and update [Clay Toggle](https://github.com/liferay/clay/blob/master/packages/clay-form/src/Toggle.tsx) to have a root `label` element , or 
1. Update the tag in this PR to match Clay markup

Personally, I think that root `label` makes a lot of sense. Is there a reason why we chose `span` over `label` in Clay?

Reply:
> In my opinion the source of truth is here. 

I think we should follow this pattern that @pat270 linked. If ClayToggle doesn't match that, an issue should be opened on the clay repo and then the tag here should match that markup as well.

I'm not totally sure why we used `span` instead of `label` for this component. My guess is that we used the 2.x component for reference, and [it looks like 2.x used span as well](https://github.com/liferay/clay/blob/2.x/packages/clay-label/src/ClayLabel.soy#L53).

Reply:
I opened https://github.com/liferay/clay/issues/3838.

As for the code in this PR, :+1: